pkg/aws/client.go

```diff
@@ -93,6 +93,8 @@ var addROSAVersionToUserAgent = request.NamedHandler{
 }
 
 // Client defines a client interface
+//
+//go:generate mockgen -source=client.go -package=mocks -destination=mocks/client.go
```

Comment:
@hunterkepley If you rebase on `master`, I think a `mock` got merged as part of #1742